src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -124,6 +124,7 @@ public static class ScriptConstants
         public const string FeatureFlagDisableWorkerIndexing = "DisableWorkerIndexing";
         public const string FeatureFlagEnableMultiLanguageWorker = "EnableMultiLanguageWorker";
         public const string FeatureFlagEnableLinuxEPExecutionCount = "EnableLinuxFEC";
+        public const string FeatureFlagEnableLogsInHostV3 = "EnableLogsInHostV3";
```

Comment:
Let's simply call this `EnableHostLogs`

Reply:
Done. Naming looks much better now. Thanks